### PR TITLE
[workers] Update new-url-parser-implementation.md

### PR DESCRIPTION
**Pull request by jclee**
Update description of the `new-url-parser-implementation` compatibility flag to provide a bit more detail and reflect the fact that it is now the current default.

### CODE REVIEW

File: content/workers/_partials/_platform-compatibility-dates/new-url-parser-implementation.md [hunk 1]
```diff
@@ -11,4 +11,4 @@ enable_flag: "url_standard"
 disable_flag: "url_original"
 ---
 
-The original Workers `URL` API implementation is not fully compliant with the [WHATWG URL Standard](https://url.spec.whatwg.org/). Cloudflare has added a new implementation that is fully compliant. However, since the new implementation is not completely backwards compatible, it is disabled by default. Use the `url_standard` flag to enable the new implementation.
+The original Workers `URL` API implementation was not fully compliant with the [WHATWG URL Standard](https://url.spec.whatwg.org/), differing in several ways, including how validation and URL encoding was performed. Use the `url_standard` flag to opt-in to the backwards-incompatible change to use a fully compliant `URL` API implementation.
```
Comment (irvinebroque) [inline content/workers/_partials/_platform-compatibility-dates/new-url-parser-implementation.md]:
```suggestion
The original implementation of the [`URL`](https://developer.mozilla.org/en-US/docs/Web/API/URL) API in Workers was not fully compliant with the [WHATWG URL Standard](https://url.spec.whatwg.org/), differing in several ways, including how validation and URL encoding was performed. Set the compatibility date of your Worker to a date after `2022-10-31` or enable the `url_standard` compatibility flag to opt-in the fully spec compliant `URL` API implementation.
```

**Review by irvinebroque — APPROVED** [review]
Thank you for updating! 🙌

**Comment by KianNH** [comment]
Is it possible to add a note that `URL.canParse(url)` is only available with the `url_standard` flag?

**Comment by jclee** [comment]
> Is it possible to add a note that `URL.canParse(url)` is only available with the `url_standard` flag?

Oh, yeah, I wasn't aware of that difference, but it makes sense if it was added after the new implementation became standard, and maybe having the text present would help searchers identify problems due to using the old implementation.  Not sure if there are other significant additions, but a cursory scan through git history isn't showing anything obvious.  Will try adding that detail, and maybe some examples of the other differences already described.

**Review by irvinebroque — APPROVED** [review]

**Comment by KianNH** [comment]
> > Is it possible to add a note that `URL.canParse(url)` is only available with the `url_standard` flag?
> 
> Oh, yeah, I wasn't aware of that difference, but it makes sense if it was added after the new implementation became standard, and maybe having the text present would help searchers identify problems due to using the old implementation. Not sure if there are other significant additions, but a cursory scan through git history isn't showing anything obvious. Will try adding that detail, and maybe some examples of the other differences already described.

Sounds good to me! The only other thing that comes to mind is the `response_redirect_url_standard` flag which opts into the same behaviour for the `Response.redirect()` static method - maybe this flag could link to that flag as well?

File: content/workers/_partials/_platform-compatibility-dates/new-url-parser-implementation.md [hunk 2]
```diff
@@ -11,4 +11,20 @@ enable_flag: "url_standard"
...
+
+* The original implementation would percent-encode or percent-decode certain content differently:
+
+  `new URL("https://example.com/a%40b?c d%20e?f").toString() === "https://example.com/a@b?c+d+e%3Ff"`
+
+* The original implementation lacked more recently implemented `URL` features, like [`URL.canParse()`](https://developer.mozilla.org/en-US/docs/Web/API/URL/canParse_static).
+
+Set the compatibility date of your Worker to a date after `2022-10-31` or enable the `url_standard` compatibility flag to opt-in the fully spec compliant `URL` API implementation.
+
+See also the [`response_redirect_url_standard` compatibility flag](/workers/configuration/compatibility-dates/#use-a-spec-compliant-url-implementation-in-redirects) , which affects the URL implementation used in `Response.redirect()`.
```
Comment (deadlypants1973) [inline content/workers/_partials/_platform-compatibility-dates/new-url-parser-implementation.md]:
```suggestion
Refer to the [`response_redirect_url_standard` compatibility flag](/workers/configuration/compatibility-dates/#use-a-spec-compliant-url-implementation-in-redirects) , which affects the URL implementation used in `Response.redirect()`.
```

**Review by deadlypants1973 — APPROVED** [review]
PCX

**Pull request merged**